okhttp/third_party/okhttp/java/io/grpc/okhttp/internal/Platform.java

```diff
@@ -105,47 +119,71 @@ public void connectSocket(Socket socket, InetSocketAddress address,
   }
 
   /** Attempt to match the host runtime to a capable Platform implementation. */
+  @SuppressWarnings("unchecked")
```

Comment:
@ejona86 Given that this has already diverged significantly I'm not overly concerned with merging as discussed.